### PR TITLE
feat(model): Creating a model to represent data

### PR DESCRIPTION
**Pull request by shraddha1704**
### What
Creating db model to represent movie data

### Why
Miyagi training

### CODE REVIEW

File: lib/models/movie.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+    tableName: 'movies',
+    serialize: function() {
```
Comment (ajpawlicki) [inline lib/models/movie.js]: nit: use arrow function

**Review by ajpawlicki — APPROVED** [review]

File: lib/models/movie.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    tableName: 'movies',
+    serialize: () => {
+      return {
+        id: this.get('id'),
+        title: this.get('title'),
+        release_year: this.get('release_year'),
+        object: 'movie'
+      };
+    }
+  });
```
Comment (mgartner) [inline lib/models/movie.js]: nit: we'd usually indent this like:

```js
module.exports = Bookshelf.Model.extend({
  tableName: 'movies',
  serialize: () => {
    return {
      id: this.get('id'),
      title: this.get('title'),
      release_year: this.get('release_year'),
      object: 'movie'
    };
  }
});
```

File: db/migrations/20190313145917_create_movies.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 exports.up = async (Knex) => {
-        await Knex.schema.createTable('movies', (table) => {
+    await Knex.schema.createTable('movies', (table) => {
```
Comment (mgartner) [inline db/migrations/20190313145917_create_movies.js:4]: Looks like this file is indenting with 4 spaces. We typically indent with 2 spaces in our Javascript repositories.

**Review by mgartner — APPROVED** [review]
LGTM, just some nits.

File: db/migrations/20190313145917_create_movies.js (hunk 3 above)
Comment (shraddha1704) [reply db/migrations/20190313145917_create_movies.js:4]: oh. i'll change the settings

**Pull request merged**